docker/build_image.sh

```diff
@@ -47,6 +50,12 @@ do
           CUDA_VERSION="cu117"
           shift
           ;;
+        -bi|--baseimage)
+          USER_BASE_IMAGE="$2"
+          UPDATE_BASE_IMAGE=true
```

Comment:
What's wrong with overwriting the base image?, I don't believe we need this extra variable

Reply:
Depending on the order of the args, it can get overwritten. Ex: `-cv` arg can overwrite `-bi`